src/main/java/school/lemon/changerequest/java/introduction/hw1/Task5.java

```diff
@@ -3,10 +3,18 @@
 
 public class Task5 {
     public static double calculateA(double x, double y, double z) {
-        return 0;
+        double a = 1/2;
```

Comment:
1. The same thing, as previously. 1/2 is 0. Because you are using two integers. In java literals are integers by default. 
You can specify, that you want to use double/float in a couple of ways: `1d/2; 1./2; 1/2.; 1/2d; 1/2.; 1.0/2.0; 1./2.; 1d/2d;`. Please read one more time about double/float and about auto casting of digits.
2. You have an error in `b` calculation. You don't need to calculate sin from `calculateB` result. Please factor out this multiplication.